genrl/deep/common/base.py

```diff
@@ -54,29 +92,63 @@ def get_action(self, state, deterministic=False):
 
 
 class BaseValue(nn.Module):
+    """
+    Basic implementation of a general Value function
+    """
     def __init__(self):
         super(BaseValue, self).__init__()
 
         self.model = None
 
     def forward(self, x):
+        """
+        Defines the computation performed at every call.
+
+        :param x: Input to value function
+        :type x: Tensor
+        """
         return self.model.forward(x)
 
     def get_value(self, x):
+        """
+        Get value from value function based on input
+
+        :param x: Input to value function
+        :type x: Tensor
+        :returns: Value
+        """
         return self.forward(x).squeeze(-1)
 
 
 class BaseActorCritic(nn.Module):
+    """
+    Basic implementation of a general Actor Critic
+    """
     def __init__(self):
         super(BaseActorCritic, self).__init__()
 
         self.actor = None
         self.critic = None
 
     def get_action(self, state, deterministic=False):
-        state = torch.as_tensor(state).float()
```

Comment:
Are you sure omitting this line is not having any effect to the agents?

Reply:
This may be the issue. Thanks for pointing it out!